packages/python/plotly/plotly/matplotlylib/mplexporter/exporter.py

```diff
@@ -285,16 +285,13 @@ def draw_collection(
             "zorder": collection.get_zorder(),
         }
 
-        offset_dict = {"data": "before", "screen": "after"}
-        offset_order = offset_dict[collection.get_offset_position()]
```

Comment:
Thanks @NeilGirdhar - in as far as `"screen"` is now the sole possibility, this is reasonable. I wonder though, for users who still have MPL 3.4 or before, what's the likelihood they have `"data"` and thus this change would break their use case? We could support this with something like:
```py
offset_order = offset_dict[
    collection.get_offset_position()
    if hasattr(collection, "get_offset_position")
    else "screen"
]
```

Reply:
Good point.  Done, as requested.